File: aicook_mobile/lib/models/Farm.dart
class Farm {
  final String id;
  final String name;
  final String description;
  final String city;
  final String address;
  final double lat;
  final double lng;

  Farm(
      {this.id,
      this.name,
      this.description,
      this.city,
      this.address,
      this.lat,
      this.lng});

  factory Farm.fromJson(Map<String, dynamic> json) {
    return Farm(
      id: json['_id'],
      name: json['name'],
      description: json['description'],
      city: json['city'],
      address: json['address'],
      lat: json['lat'].toDouble(),
      lng: json['lng'].toDouble(),
    );
  }
}

File: aicook_api/lib/models/farm.js
var mongoose = require('mongoose')

var FarmSchema = new mongoose.Schema({
    "name": {
        required: true,
        type: String
    },
    "descrition": String,
    "image": String,

    "city": String,
    "zipcode": String,
    "address": String,
    "lat": Number,
    "lng": Number,

    products: [{
        "name": String,
        "description": String,
        "price": Number,
    }]
})

var Farm = mongoose.model('Farm', FarmSchema)
module.exports = {
    Farm
}
// "name": ,
// "descrition": ,

// "city": ,
// "zipcode": ,
// "address": ,
// "lat": ,
// "long": ,

// products: [{
//     "name": ,
//     "price": ,
//     "description": ,
// }]